lib/templates/webmanifest.js

```diff
@@ -2,12 +2,16 @@ const renderWebManifest = (name, color) => `
 {
 	"name": "Gradebook",
 	"short_name": "Gradebook",
+	"description": "The only free app built by students to simplify grade tracking. Let's all get a little better at keeping track of our grades, together.",
```

Comment:
```suggestion
	"description": "Better grades, together. Gradebook is the only student-built, student-centered Grade Tracking app designed to help you take control of your grades",
```

We should use the same description as the marketing site